sdk/newrelic/Azure.ResourceManager.Newrelic/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Please replace the `Unreleased` with the expected release date of this first beta

Reply:
Done.